kotlin-popper/src/main/generated/popper/core/Padding.kt

```diff
@@ -0,0 +1,5 @@
+// Automatically generated - do not modify!
+
+package popper.core
+
+typealias Padding = SideObject // number | Partial<SideObject>
```

Comment:
it seems it is too strict limitations because original type is `Partial<SideObject>`